events/scheme_events/stasis_trap_scheme/wc_stasis_trap_outcome_events.txt

```diff
@@ -0,0 +1,351 @@
+		ai_chance = {
+			base = 100
+		}
+	}
+
+	option = {
+		name = stasis_trap_outcome.1001.b
+		custom_tooltip = do_not_execute_stasis_trap_tooltip
+
+		dont_excecute_scheme_effect = yes
+
+		ai_chance = {
+			base = 0
+		}
+	}
+}
+
+#####################################
+# SUCCESS OUTCOMES 2001 - 2999
+#####################################
+
+#######################
+# Trapped Residence
+#######################
+stasis_trap_outcome.2001 = {
+	type = character_event
+	title = stasis_trap_outcome.1001.t
+	desc = {
+		desc = stasis_trap_outcome.2001.desc
+
+		first_valid = {
+			# If the Scheme Owner is known:
+			triggered_desc = {
+				trigger = { 
+					OR = {
+						AND = {
+							exists = scope:scheme_discovered
+							scope:scheme_discovered = yes 
+						}
+						scope:scheme = {
+							is_scheme_exposed = yes
+						}
+					}
+				}
+				desc = stasis_trap_outcome.2001.owner_is_known
+			}
+			desc = stasis_trap_outcome.2001.no_awareness
+		}
+	}
+	theme = intrigue
+	left_portrait = {
+		character = scope:target
+		animation = fear
+	}
+	widget = {
+		gui = "event_window_widget_scheme"
+		container = "custom_widgets_container"
+	}
+	
+	trigger = {
+		scope:stasis_trap_method = flag:trapped_residence
+	}
+
+	immediate = {
+		if = {
+			limit = {
+				exists = scope:scheme_discovered
+				scope:scheme_discovered = yes 
+			}
+
+			stasis_trap_success_effect = { DISCOVERED = scope:scheme_discovered }
+		}
+		else = {
+			stasis_trap_success_effect = { DISCOVERED = no }
+		}
+		
+	}
+
+	option = {
+		name = stasis_trap_outcome.2001.a
+		play_music_cue = "mx_cue_prison"
+		successful_stasis_trap_outcome_event_option_effect = yes
+	}
+}
+
+#################################
+# FAILURE OUTCOMES 4000 - 4999
+#################################
+stasis_trap_outcome.4001 = {
+	type = character_event
+	title = stasis_trap_outcome.1001.t
+	desc = {
+		desc = stasis_trap_outcome.4001.opening
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					exists = scope:scheme_discovered
+				}
+				desc = stasis_trap_outcome.4001.owner_discovered
+			}
+			desc = stasis_trap_outcome.4001.owner_secret
+		}
+	}
+	theme = intrigue
+	left_portrait = {
+		character = scope:target
+		animation = rage
+	}
+	widget = {
+		gui = "event_window_widget_scheme"
+		container = "custom_widgets_container"
+	}
+	
+	trigger = {
+		scope:stasis_trap_method = flag:trapped_residence
+	}
+
+	immediate = {
+		stasis_trap_failure_effect = yes
+	}
+
+	option = { # Abandon scheme
+		name = stasis_trap_outcome.4001.a
+		abandon_scheme_effect = yes
+
+		ai_chance = {
+			base = 100
+		}
+	}
+
+	option = { # Try again!
+		name = stasis_trap_outcome.4001.b
+		restart_scheme_effect = yes
+
+		ai_chance = {
+			base = 10
+			modifier = {
+				add = 200
+				has_trait = stubborn
+			}
+		}
+	}
+}
+
+########################################
+# TARGET SUCCESS NOTIFICATIONS
+########################################
+
+#I have been stunned!
+stasis_trap_outcome.5001 = {
```

Comment:
This text doesn't end with a dot.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117648426-3f9b7080-b19f-11eb-832c-019ce0f1158e.png)

</details>